src/schema/v2/conversation/collectorResume.ts

```diff
@@ -14,5 +19,10 @@ export const CollectorResume = new GraphQLObjectType<any, ResolverContext>({
       description: "The Collector follows the Gallery profile",
       resolve: ({ isCollectorFollowingPartner }) => isCollectorFollowingPartner,
     },
+    userId: {
```

Comment:
This should resolve a real user, not just the ID

Reply:
Actually this is correct. You can't resolve a real user, b/c partners won't have access to fetch a user - that's why they use this collector resume/profile thing.

This id is solely used for stitching purposes (to stitch in the buyer activity data). Maybe the description here can mention that?

Reply:
Because in our stitching setup, you need to 'dependency inject or bootstrap' any data onto the parent, so you can refer to that in the stitched resolver. So this pattern, where we _add_ a field to the schema which is intended to be used _just_ in the stitching resolver for another field, is something we do. Sometimes we prefix it with `_do_not_use` or make it clear via a comment or `description` that this is just used for stitching purposes.

Reply:
I'll add the description for this field!